doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+
+.. _gs_cxx_requests_close:
+
+Closing connections
+~~~~~~~~~~~~~~~~~~~~
+
+Finally, a user is responsible for closing connections.
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel12-1
+   :end-before: doclabel12-2
+   :language: c
+
+.. _gs_cxx_build:
+
+Building and launching C++ application
+-------------------------------------------
+
+Now, we are going to build our example C++ application, launch it
+to connect to Tarantool instance and execute all the requests defined.
+
+Make sure you are in the root directory of the cloned C++ connector repository.
+To build the example application:
+
+.. code-block:: bash
+
+   cd examples
+   cmake .
+   make
+
+Make sure the :ref:`Tarantool session <gs_cxx_prereq_tnt_run>`
+you started earlier is running. Launch the application:
+
+.. code-block:: bash
+
+   ./Simple
+
+As you can see from the execution log, all the connections to Tarantool
+defined in our application have been established and all the requests
+have been executed successfully.
+
+.. //TBD Maybe to quote here the full execution log (62 lines)
+
+.. _gs_cxx_reader:
+
+Decoding and reading the data
+------------------------------
+
+Responses from a Tarantool instance contain raw data, that is, the data encoded
+into the msgpack tuples. To decode client's data, a user has to write one's own
+decoders (readers) based on the database schema and include them in one's
+application:
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel01-3
+   :end-before: doclabel01-4
+   :language: c
+
+To show the logic of decoding a response, we will use
+`the reader from our example <https://github.com/tarantool/tntcxx/blob/master/examples/Reader.hpp>`_.
+
+First, the structure corresponding our :ref:`example space format <gs_cxx_prereq_tnt_run>`
+is defined:
+
+.. literalinclude:: ./_includes/Reader.cpp
+   :start-after: doclabel13-1
+   :end-before: doclabel13-2
+   :language: c
+
+.. _gs_cxx_reader_base:
+
+Base reader prototype
+~~~~~~~~~~~~~~~~~~~~~~
+
+Prototype of the base reader is given in ``src/mpp/Dec.hpp``:
+
+.. code-block:: c
+
+   template <class BUFFER, Type TYPE>
+   struct SimpleReaderBase : DefaultErrorHandler {
+       using BufferIterator_t = typename BUFFER::iterator;
+       /* Allowed type of values to be parsed. */
+       static constexpr Type VALID_TYPES = TYPE;
+       BufferIterator_t* StoreEndIterator() { return nullptr; }
+   };
+
+Every new reader should inherit from it or directly from the
+``DefaultErrorHandler``.
+
+.. _gs_cxx_reader_parse_value:
+
+Parsing values
+~~~~~~~~~~~~~~~
+
+To parse a particular value, we should define the ``Value()`` method.
+First two arguments of the method are common and unused as a rule,
+but the third one defines the parsed value. In case of POD structures, it's
```

Comment:
What's POD? A link would be helpful.

Reply:
Plain Old Data. Added the link to Wikipedia article.